resources/intent/fokoos_base/tpu/fokoos_tpu_nozzle_0.40_layer_0.20_normal.inst.cfg

```diff
@@ -0,0 +1,51 @@
+[general]
+version = 4
+name = Normal
+definition = fokoos_base
+
+[metadata]
+setting_version = 20
+type = intent
+intent_category = FKS Fast
+quality_type = fokoos_layer_020
+material = generic_tpu
+variant = 0.40mm_fokoos_nozzle 
+
+[values]
+layer_height_0 = =0.3
+
+wall_thickness = =line_width*2
+wall_0_wipe_dist = 0.2
+z_seam_type = sharpest_corner 
+
+top_bottom_thickness = =layer_height*3
+
+infill_pattern = ="grid"
+infill_overlap = 10
+infill_before_walls = true
+
+material_print_temperature = 225
```

Comment:
This is not a good idea... If a material print temperature is set in a quality / intent, it can't be changed by changing the value in the material. You might want to have a look at how Ultimaker profiles handle this.

Reply:
i got it